pkg/consuldp/consul_dataplane.go

```diff
@@ -216,6 +221,14 @@ func (cdp *ConsulDataplane) Run(ctx context.Context) error {
 		return err
 	}
 
+	cdp.hcpTelemetry = telemetry.NewHCPExporter(
```

Comment:
It doesn't look like we use this pointer anywhere else. Any reason not to do this?
```go
go telemetry.NewHCPExporter(...).Run(ctx)
```

Reply:
That makes sense to me, just switched. I debated just changing the signature to `go telemetry.Run(ctx, ...params)`, might still do that...